internal/kubernetes/operator/project/project.go

```diff
@@ -70,12 +70,7 @@ type AtlasProjectResult struct {
 	Teams   []*akov2.AtlasTeam
 }
 
-func BuildAtlasProject(projectStore store.OperatorProjectStore, validator features.FeatureValidator, orgID, projectID, targetNamespace string, includeSecret bool, dictionary map[string]string, version string) (*AtlasProjectResult, error) { //nolint:gocyclo
-	project, err := projectStore.Project(projectID)
-	if err != nil {
-		return nil, err
-	}
-
+func BuildAtlasProject(projectStore store.OperatorProjectStore, project *atlasv2.Group, validator features.FeatureValidator, orgID, projectID, targetNamespace string, includeSecret bool, dictionary map[string]string, version string) (*AtlasProjectResult, error) { //nolint:gocyclo
```

Comment:
if you agree, it's a good candidate for a request struct instead of a lot of parameters

Reply:
it might make sense, yes

Reply:
Done